torch/_dynamo/utils.py

```diff
@@ -985,3 +987,102 @@ def _get_debug_dir(root_dir):
 def get_debug_dir():
     debug_root = config.debug_dir_root
     return _get_debug_dir(debug_root)
+
+
+def get_fake_value(node, tx):
```

Comment:
any changes to this or just code motion?

Reply:
motion